File: year2020/month09/day0924/ms_0924.cpp
#include <iostream>

using namespace std;

void sumOfAbs(int* p);

int main() {
	int data[5];

	for (int i = 0; i < 5; i++)
		scanf_s("%d", &data[i]);

	sumOfAbs(data);

	return 0;
}

void sumOfAbs(int* p) {
	int sum = 0;

	for (int i = 0; i < 5; i++, p++)
		sum += abs(*p);

	cout << sum << endl;
}

File: year2020/month09/day0924/ms_0924.cs
using System;

namespace algorithm_63
{
    class ms_0924
    {
        static void SumOfAbs(int[] data) 
        {
            int sum = 0;

            for(int i = 0; i < data.Length; i++)
                sum += Math.Abs(data[i]);

            Console.WriteLine(sum);
        }

        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int[] data = new int[5];

            for(int i = 0; i < data.Length; i++)
                data[i] = Convert.ToInt32(input[i]);
                
            SumOfAbs(data);
        }
    }
}
